### ISSUE TITLE
Object property global override

### ISSUE THREAD

**Issue by AlexanderAllen** [AlexanderAllen/panettone#29]
TL;DR: **Combining nullable with default values provides a form of fault tolerance and grace when using DTOs for inconsistent services.**

_Sometimes_... just sometimes you have a 10,000 line long Open API schema that you'd like to generate types from (DTOs), but none or seldom any of the properties are set as nullable or specify a default value. Or maybe they are inconsistent and you have to go through the source in a treasure hunt. Maybe you'd like to leave the source alone, untouched, and rely on a type generator to do you some small favors...

When using DTOs to consume data from a service, a type without a default value will require that the service response always includes data for the property without a default value. This becomes a problem when the service _does not_ reply with a value for such non-default properties, leading to bugs and/or additional service response handling code.

Nullable on the other hand will allow object properties to be represented as null, which is a benefit because it can go hand in hand with default values, when setting default property values to null.

A global override for nullable + default value (set to null) side steps the issue of having through a long OAS changing every single property to either or both.

The same kind of feature request can be applied to all other kinds of global property configuration. This ticket is **just** for those two prop modifiers, and maybe and third one, and that's it. Any other go in a separate ticket as everything needs to be unit tested.
